sw/host/cryptotest/testvectors/data/schemas/aes_kw_schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/lowRISC/opentitan/master/sw/host/cryptotest/testvectors/data/schemas/aes_kw_schema.json",
+  "title": "Cryptotest AES-KWP Testvector",
+  "description": "A list of testvectors for AES-KWP testing",
+  "$defs": {
+    "byte_array": {
+      "type": "array",
+      "items": {
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 255
+      }
+    }
+  },
+  "type": "array",
+  "items": {
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "test_case_id": {
```

Comment:
Along with the test case ID, I usually include a `vendor` field, with one of the values `[nist, wycheproof, manual, ...]`.